### PR TITLE
[docs](docs) update en docs

### PR DESCRIPTION
**Pull request by httpshirley**
更新了“数据湖文档”章节[文件分析](https://doris.apache.org/zh-CN/docs/dev/lakehouse/file)与[JDBC外表](https://doris.apache.org/zh-CN/docs/dev/lakehouse/external-table/jdbc)的对应英文文档

# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [x] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by luzhijing** [comment]
LGTM

**Comment by jialiangCHOU** [comment]
LGTM

**Comment by luzhijing** [comment]
LGTM

**Review by hf200012 — APPROVED** [review]
LGTM

File: docs/en/docs/lakehouse/external-table/jdbc.md [hunk 1]
```diff
@@ -28,7 +28,250 @@ under the License.
 
 <version deprecated="1.2.2">
 
-Please use [JDBC Catalog](../multi-catalog/jdbc) to visit JDBC data source.
+Please use [JDBC Catalog](https://doris.apache.org/docs/dev/lakehouse/multi-catalog/jdbc/) to access JDBC data sources.
```
Comment (morningman) [inline docs/en/docs/lakehouse/external-table/jdbc.md:31]: Use relative path

File: docs/en/docs/lakehouse/external-table/jdbc.md [hunk 2]
```diff
@@ -28,7 +28,250 @@ under the License.
...
+1. Multiple data sources can be connected to Doris;
+2. It enables Join queries across Doris and other data sources and thus allows more complex analysis.
+
+This topic introduces how to use JDBC External Tables in Doris.
+
+</version>
+
+### Create JDBC External Table in Doris
+
+See [CREATE TABLE](https://doris.apache.org/docs/dev/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE/) for syntax details.
```
Comment (morningman) [inline docs/en/docs/lakehouse/external-table/jdbc.md:48]: use relative path

File: docs/en/docs/lakehouse/external-table/jdbc.md [hunk 3]
```diff
@@ -28,7 +28,250 @@ under the License.
...
+|  DECIMAL   | DECIMAL  |
+
+Note:
+
+- Some data types in ClickHouse, such as UUID, IPv4, IPv6, and Enum8, will be mapped to Varchar/String in Doris. IPv4 and IPv6 will be displayed with an `/` as a prefix. You can use the `split_part` function to remove the `/` .
+- The Point Geo type in ClickHouse cannot be mapped in Doris by far. 
+
+## Q&A
+
+See the FAQ section in [JDBC](https://doris.apache.org/docs/dev/lakehouse/multi-catalog/jdbc/).
```
Comment (morningman) [inline docs/en/docs/lakehouse/external-table/jdbc.md:276]: Use relative path

File: docs/en/docs/lakehouse/file.md [hunk 4]
```diff
@@ -27,5 +27,114 @@ under the License.
...
+
+With the Table Value Function feature, Doris is able to query files in object storage or HDFS as simply as querying Tables. In addition, it supports automatic column type inference.
+
+</version>
+
+## Usage
+
+For more usage details, please see the documentation:
+
+* [S3](https://doris.apache.org/docs/dev/sql-manual/sql-functions/table-functions/s3/): supports file analysis on object storage compatible with S3
```
Comment (morningman) [inline docs/en/docs/lakehouse/file.md:40]: ditto

**Pull request merged**